### PR TITLE
DLP-1890: add support for Risk Score Integrations

### PR DESCRIPTION
**Pull request by MattDavis00**
https://developers.cloudflare.com/api/operations/dlp-zt-risk-score-integration-create

## Description

Adds support for the Risk Score Integrations API to the CF Go library.

## Has your change been tested?

Unit tests. Have also tested these changes alongside my terraform-provider-cloudflare changes to check it works.

## Types of changes

What sort of change does your code introduce/modify?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [X] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

**Comment by jacobbednarz** [comment]
this already exists on v2 - https://github.com/cloudflare/cloudflare-go/blob/v2/zero_trust/riskscoringintegration.go . do you want it in both versions?

**Comment by MattDavis00** [comment]
> this already exists on v2 - https://github.com/cloudflare/cloudflare-go/blob/v2/zero_trust/riskscoringintegration.go . do you want it in both versions?

Oh damn I didn't realise, nice. Yeah lets add this to v1 as well

File: zt_risk_score_integration.go [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+	IntegrationType string     `json:"integration_type,omitempty"`
+	TenantUrl       string     `json:"tenant_url,omitempty"`
+	ReferenceID     string     `json:"reference_id,omitempty"`
+	AccountTag      string     `json:"account_tag,omitempty"`
+	WellKnownUrl    string     `json:"well_known_url,omitempty"`
+	Active          *bool      `json:"active,omitempty"`
+	CreatedAt       *time.Time `json:"created_at,omitempty"`
+}
+
+// The properties required to create a new risk score integration
```
Comment (jacobbednarz) [inline zt_risk_score_integration.go]:
```suggestion
// The properties required to create a new risk score integration.
```

File: zt_risk_score_integration.go [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+type RiskScoreIntegrationCreateRequest struct {
+	IntegrationType string `json:"integration_type,omitempty"`
+	TenantUrl       string `json:"tenant_url,omitempty"`
+	ReferenceID     string `json:"reference_id,omitempty"`
+}
+
+// The properties required to update a risk score integration
```
Comment (jacobbednarz) [inline zt_risk_score_integration.go]:
```suggestion
// The properties required to update a risk score integration.
```

**Pull request merged**